### PR TITLE
Converting list of date and time placeholders into a table

### PR DESCRIPTION
**Pull request by frommMoritz**

### CODE REVIEW

**Comment by Maikuolan** [comment]
@frommMoritz : Nice idea, and thanks for the contribution! This should help users to better understand the placeholders, I think. :+1: 😊

I'll leave this PR open for a few hours or so in case anyone has any comments, and otherwise will merge after that.

( @DanielRuf : If you've got a spare moment, could you check the spelling and grammar for the changed lines? Cheers. 😊 )

**Comment by DanielRuf** [comment]
Will do so.

File: _docs/readme.de.md [hunk 1]
```diff
@@ -329,11 +329,21 @@ Generelle Konfiguration von CIDRAM.
 - Name einer Apache-Stil-Datei zu protokollieren alle blockierten Zugriffsversuche. Geben Sie einen Dateinamen an oder lassen Sie die Option zum Deaktivieren leer.
 
 ##### "logfileSerialized"
-- Name einer Datei zu protokollieren alle blockierten Zugriffsversuche (Format ist serialisiert). Geben Sie einen Dateinamen an oder lassen Sie die Option zum Deaktivieren leer.
+- Name einer Datei in welcher serialisiert alle blokierten zugriffsversuche Protokolliert werden. Geben Sie einen Dateinamen an oder lassen Sie die Option zum Deaktivieren leer.
```
Comment (DanielRuf) [inline _docs/readme.de.md]: zugriffsversuche => Zugriffsversuche
Protokolliert => protokolliert
Deaktivieren => deaktivieren

File: _docs/readme.de.md [hunk 2]
```diff
@@ -329,11 +329,21 @@ Generelle Konfiguration von CIDRAM.
...
 
-*Nützlicher Tipp: Wenn du willst, Sie können Datum/Uhrzeit-Information den Namen der Protokolldateien anhängen, durch diese im Namen einschließlich: `{yyyy}` für die komplette Jahr, `{yy}` für abgekürzte Jahr, `{mm}` für Monate, `{dd}` für Tag, `{hh}` für Stunde.*
+*Nützlicher Tipp: Wenn Sie wollen, können Sie Datum/Uhrzeit-Information im Namen der Protokolldateien verwenend.*
```
Comment (DanielRuf) [inline _docs/readme.de.md]: verwenend => verwenden

File: _docs/readme.de.md [hunk 3]
```diff
@@ -329,11 +329,21 @@ Generelle Konfiguration von CIDRAM.
...
 
-*Beispielen:*
+Platzhalter|Beschreibung|Beispiel
+--- | --- | ---
+`{yyyy}`|Komplettes  Jahr|2018
```
Comment (DanielRuf) [inline _docs/readme.de.md]: Komplettes => komplettes

File: _docs/readme.de.md [hunk 4]
```diff
@@ -329,11 +329,21 @@ Generelle Konfiguration von CIDRAM.
...
+`{yy}`|Abgekürtztes Jahr|18
```
Comment (DanielRuf) [inline _docs/readme.de.md]: Abgekürtztes => abgekürztes

File: _docs/readme.en.md [hunk 5]
```diff
@@ -331,7 +331,17 @@ General CIDRAM configuration.
 ##### "logfileSerialized"
 - Serialised file for logging all blocked access attempts. Specify a filename, or leave blank to disable.
 
-*Useful tip: If you want, you can append date/time information to the names of your logfiles by including these in the name: `{yyyy}` for complete year, `{yy}` for abbreviated year, `{mm}` for month, `{dd}` for day, `{hh}` for hour.*
+*Useful tip: If you want, you can append date/time information to the names of your logfiles by including these in the name.*
+
+Placeholder|Explanation|Example
+--- | --- | ---
+`{yyyy}`|Entire Year|2018
```
Comment (DanielRuf) [inline _docs/readme.en.md:338]: Entire => entire
Year => year

File: _docs/readme.en.md [hunk 6]
```diff
@@ -331,7 +331,17 @@ General CIDRAM configuration.
...
+`{yy}`|Shortened Year|18
```
Comment (DanielRuf) [inline _docs/readme.en.md:339]: Shortened => shortened
Year => year

Comment (DanielRuf) [reply _docs/readme.en.md:339]: We should check for lowercase / capitalized and not mi these.

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: _docs/readme.de.md [hunk 7]
```diff
@@ -329,21 +329,11 @@ Generelle Konfiguration von CIDRAM.
...
-`{yy}`|Abgekürtztes Jahr|18
-`{mm}`|Monat|08
-`{dd}`|Tag|20
-`{hh}`|Stunde|12
-
-_Beispiele anhand des 20.08.2018 um 12:06_
+*Nützlicher Tipp: Wenn du willst, Sie können Datum/Uhrzeit-Information den Namen der Protokolldateien anhängen, durch diese im Namen einschließlich: `{yyyy}` für die komplette Jahr, `{yy}` für abgekürzte Jahr, `{mm}` für Monate, `{dd}` für Tag, `{hh}` für Stunde.*
 
-*Beispiele:*
+*Beispielen:*
```
Comment (DanielRuf) [inline _docs/readme.de.md]: Beispielen => Beispiele

File: _docs/readme.de.md [hunk 8]
```diff
@@ -329,21 +329,11 @@ Generelle Konfiguration von CIDRAM.
...
-Platzhalter|Beschreibung|Beispiel
---- | --- | ---
-`{yyyy}`|Komplettes  Jahr|2018
-`{yy}`|Abgekürtztes Jahr|18
-`{mm}`|Monat|08
-`{dd}`|Tag|20
-`{hh}`|Stunde|12
-
-_Beispiele anhand des 20.08.2018 um 12:06_
+*Nützlicher Tipp: Wenn du willst, Sie können Datum/Uhrzeit-Information den Namen der Protokolldateien anhängen, durch diese im Namen einschließlich: `{yyyy}` für die komplette Jahr, `{yy}` für abgekürzte Jahr, `{mm}` für Monate, `{dd}` für Tag, `{hh}` für Stunde.*
```
Comment (DanielRuf) [inline _docs/readme.de.md]: Totally incorrect wordings, please check again.

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by Maikuolan** [comment]
Because English and German translations are now separated into two distinct pull requests, and this pull request now contains only the changes to the English README, and because the requested changes to the English version are relatively quick and easy to fix (correct capitalisation, etc), I'll go ahead and merge this now (I can fix the capitalisation myself a little later tonight).

**Comment by Maikuolan** [comment]
*Note: Requested changes for the German README are still relevant, but apply now to #96.*

**Pull request merged**